program/pimoroni

```diff
@@ -677,14 +678,15 @@ MAINLIST="\
         return 0
     elif [ $RETURN -eq 0 ]; then
         case "$PHAT" in
-            1) get_install envirophat ;;
-            2) get_install iotphat ;;
-            3) get_install explorerhat ;;
-            4) get_install microdotphat ;;
-            5) get_install motephat ;;
-            6) get_install phatdac ;;
-            7) get_install scrollphat ;;
-            8) get_install unicornhat ;;
+            1) get_install automationhat ;;
```

Comment:
This is the SECOND of the (minimum) TWO occurrences that needs to be added.

again, note that other entries will be affected by addition - they NEED to correspond to the 'trigger' (number) set in the menu listing commented on previously.

... get_install is the current function call but it's likely to change when handling of logfile is added, pointing the save_stdout wrapping function (or something else if i gets implemented differently)